test/runtests.bat

```diff
@@ -6,8 +6,20 @@ if not exist ..\bin mkdir ..\bin
 REM compile the code into the bin folder
 javac  ..\src\seedu\addressbook\Addressbook.java -d ..\bin
 
+REM (invalid) no parent directory, invalid filename with no extension
+java -classpath ..\bin seedu.addressbook.AddressBook " " < NUL > actual.txt
+REM (invalid) invalid parent directory that does not exist, valid filename
+java -classpath ..\bin seedu.addressbook.AddressBook "directoryThatDoesNotExist/valid.filename" < NUL >> actual.txt
+REM (invalid) no parent directory, invalid filename with dot on first character
+java -classpath ..\bin seedu.addressbook.AddressBook ".noFilename" < NUL >> actual.txt
+REM (invalid) valid parent directory, non regular file
+mkdir data\notRegularFile.txt
```

Comment:
So I tested on Windows, and we will get
```
A subdirectory or file data\notRegularFile.txt already exists.
```
on subsequent runs.

Maybe change this to
```bat
if not exist data\notRegularFile.txt mkdir data\notRegularFile.txt
```
?